core/src/main/kotlin/com/imashnake/animite/core/ui/ProgressIndicator.kt

```diff
@@ -8,16 +8,15 @@ import androidx.compose.material3.MaterialTheme
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.clip
-import androidx.compose.ui.res.dimensionResource
-import com.imashnake.animite.R as Res
+import androidx.compose.ui.unit.dp
 
 @Composable
 fun ProgressIndicator() {
     LinearProgressIndicator(
         modifier = Modifier
             .clip(CircleShape)
-            .width(dimensionResource(Res.dimen.progress_indicator_width))
-            .height(dimensionResource(Res.dimen.progress_indicator_height)),
+            .width(100.dp)
```

Comment:
can we at least bring the dimensions into this module?

Reply:
Can do, good catch!